internal/testing/alldeps

```diff
@@ -61,6 +61,7 @@ github.com/mitchellh/go-homedir
 github.com/mitchellh/go-testing-interface
 github.com/mitchellh/mapstructure
 github.com/nats-io/go-nats
```

Comment:
fyi maybe can remove this entry here: `github.com/nats-io/go-nats` since replaced by the one below

Reply:
Thanks for the idea. This file is automatically generated to include all of the dependencies across our whole repo; the old version `go-nats` is still included due to the `samples/` module, which includes `gocloud.dev v0.15.0`, which includes the old `go-nats`. Multi-module repos are tricky, and we are still figuring out the best way to handle it.